maud_macros/src/parse.rs

```diff
@@ -88,6 +81,12 @@ impl<'cx, 'a, 'i> Parser<'cx, 'a, 'i> {
         self.input = &self.input[n..];
     }
 
+    /// Returns error message attached to sp and stops compilation immediately.
+   fn error<T>(&self, span: Span, message: &'static str) -> ParseResult<T> {
+    self.cx.span_err(span, message);
+    Err(())
+   }
```

Comment:
The indentation looks wrong here. Please make sure that all lines of code are indented by a multiple of 4 spaces.